tests/test_odxtools.py

```diff
@@ -90,16 +91,28 @@ def test_encode_response_with_matching_request_param_and_structure(self) -> None
         self.assertEqual(bytes(coded_response), bytes.fromhex("faba03"))
 
 
+@dataclass
+class X:
+    short_name: str
+    value: int
+
+
 class TestNamedItemList(unittest.TestCase):
 
+    def setUp(self) -> None:
+        self.foo_obj = NamedItemList([X("hello", 0), X("world", 1)])
+
     def test_NamedItemList(self) -> None:
+        # Test with the original object
+        foo = self.foo_obj.__class__(self.foo_obj)
```

Comment:
why are you doing this?

Reply:
I updated the existing NamedItemList unit test to run on both the original object and the pickled one to ensure that the object is correctly reconstructed. I agree that copying the object is unnecessary, so I'll update it.